src/js/actions/mask.js

```diff
@@ -142,6 +150,98 @@ define(function (require, exports) {
             menuActions.native]
     };
 
+    /**
+     * Create an circular mask the size of the currently selected layer
+     *
+     * @return {Promise}
+     */
+    var applyRectangle = function () {
```

Comment:
I don't think I like the action names. They don't exactly describe what's going on. Maybe `makeRectangleMask` and `makeEllipseMask` would be better? What do you think?